### ISSUE TITLE
Paramount+ doesn't seem to be including the Super Bowl in the schedule?

### ISSUE THREAD

**Issue by riokitumbra** [tonywagner/EPlusTV#92]
With version 2.1.9 and setting "PARAMOUNTPLUS" to true and completing login (and ESPNPLUS set to false, but the same Paramount channels are returned either way), it appears that the Super Bowl isn't being included in the schedule that gets built. Here are the logs that are returned after completing Paramount+ login:
```
=== Getting events ===
Adding event:  Bologna vs. Lecce
Adding event:  Monza vs. Hellas Verona
Adding event:  Genoa vs. Atalanta
Adding event:  AC Milan vs. Napoli
Adding event:  Riestra vs. River Plate
Adding event:  Juventus vs. Udinese
Adding event:  Newell's Old Boys vs. Racing
Adding event:  Shandong Taishan vs. Kawasaki Frontale
Adding event:  UCL Multicast
Adding event:  Copenhagen vs. Man. City
Adding event:  RB Leipzig vs. Real Madrid
Adding event:  San Lorenzo vs. Estudiantes
Adding event:  Independiente vs. Rosario Central
=== Done getting events ===
=== Building the schedule ===
Scheduling 13 entries...
=== Done building the schedule ===
=== Starting Server ===
Server started on port 8000
```
I've only used EPlusTV once before, but I think the Super Bowl should be included in the list of events. And I don't think those events aren't supposed to be included if I'm reading the paramount provider correctly, as those aren't of the types specified in "ALLOWED_LOCAL_SPORTS". Is there something that I might be missing or misconfiguring?

**Comment by m0ngr31** [comment]
Hey @riokitumbra, sorry if it didn't pull the SB for you! I wish I saw this earlier. I wonder if different regions had different titles or something and that's why it didn't catch it or something? I'll have to be more diligent about checking that in the future.

**Comment by riokitumbra** [comment]
That's alright, it wasn't critical. I'm in the US (Maryland), so I'm not sure if it was a region difference. My best guess is that it didn't see the Super Bowl because it wasn't listed in the "Live and Upcoming" section. It wasn't listed in that section on paramountplus.com, or in the multi-channel-collection/live-and-upcoming.json endpoint that is being checked. I had modified the Paramount handler to log the json file output, but it wasn't anywhere in there. I had thought the output would be saved in my Docker logs, but it seems the logs cleared (probably when I shut the Docker instance down), so unfortunately I'm not able to upload it here.

I tried to find what other endpoints might be available so that I could check them and then use the endpoint that included the game, but I couldn't find a listing of the endpoints anywhere. And I didn't have a quick way to scrape the web requests from my Android TV box to try to find them that way.
